src/OpenFOAM/containers/Lists/gpuList/textures.H

```diff
@@ -31,18 +34,25 @@ public:
 
     void destroy()
     {
+#if __CUDA_ARCH__ >= 300
         cudaDestroyTextureObject(tex);
+#else
+        delete tex;
```

Comment:
Two errors here:
- this class is a simple wrapper and cannot be freeing any memory
- even if deleting pointer here was appropriate, pointers to GPU memory need to dealocated using CUDA API.

To correct: remove the 'delete' call.